components/gcp-click-to-deploy/README.md

```diff
@@ -23,12 +23,18 @@ in the web app and used in requests to GCP services when needed.
 
 ## Development
 
-You can run the app locally for development/testing
-
+You can run the app locally for development/testing.
 ```
 cd components/gcp-click-to-deploy
 # Install dependencies
 npm install
+```
+
+Before starting the server, make a config file `user_config/app-config.yaml` under `src/` with the content from
```

Comment:
@jlewi @kunmingg why is this needed? Why can't the file by symlinked, or just loaded from that location?
Also, the location is outdated.

Reply:
So is the deployer's url. This is currently the only place I can find easily that points to the deployer.